src/components/ui/Popup/Popup.test.tsx

```diff
@@ -0,0 +1,41 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import Popup from './Popup';
+
+describe('Popup', () => {
+  const args = {
+    bottomText: 'bottom',
+    topText: 'top',
+    trigger: 'trigger',
+    goTo: '/home',
+  };
+
+  // TODO Complete test
```

Comment:
Let's resolve this

Reply:
I cannot use `userEvent.hover` to make top and bottomtext appear, should we just discard the test?

Reply:
Yes